specs/gcc.anod

```diff
@@ -113,7 +141,11 @@ class GCC(spec('common')):
                 self.deps[m].setenv()
 
         # We need to install binutils in the compiler package
-        self.deps["binutils"].merge(self["INSTALL_DIR"])
+        if "binutils" in self.deps:
```

Comment:
So that means there are no binutils packaged with GNAT.
Was it like that the GNAT Community release? @setton